sdk/ai/azure-ai-agents/README.md

```diff
@@ -1169,7 +1169,7 @@ from opentelemetry import trace
 from azure.monitor.opentelemetry import configure_azure_monitor
 
 # Enable Azure Monitor tracing
-application_insights_connection_string = os.environ["AI_APPINSIGHTS_CONNECTION_STRING"]
+application_insights_connection_string = os.environ["APPLICATIONINSIGHTS_CONNECTION_STRING"]
```

Comment:
this is an official AzMon connection string env var, well documented and widely used - https://learn.microsoft.com/en-us/azure/azure-monitor/app/connection-strings#when-do-i-need-to-set-the-environment-variable

let's stick to it everywhere